docs/current/guides/snippets/use-secrets/host-fs/main.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+import anyio
+import dagger
+
+async def main():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+        # read file
+        with open("/home/USER/.config/gh/hosts.yml") as file:
+          f = file.read()
```

Comment:
This is blocking code. No problem in the context of this script, but best show good patterns:
```suggestion
        config = await anyio.Path("/home/USER/.config/gh/hosts.yml").read_text()
```